src/modules/fancyzones/editor/FancyZonesEditor/loc/nl/src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx.lcl

```diff
@@ -409,6 +409,24 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
+      <Item ItemId=";KeyboardControlsDescription" ItemType="0;.resx" PsrId="211" Leaf="true">
+        <Str Cat="Text">
+          <Val><![CDATA[]D;]A;     - [Shift]5D;+S to split currently focused zone.]D;]A;     - Ctrl+Tab to focus zones/resizers.]D;]A;     - Tab to cycle zones and resizers.]D;]A;     - Delete to remove the focused resizer.]D;]A;     - Arrows to move the focused resizer.]]></Val>
+          <Tgt Cat="Text" Stat="Loc" Orig="New">
+            <Val><![CDATA[]D;]A;     - [Shift]5D;+S om de zone in focus te splitsen.]D;]A;     - Ctrl+Tab om zones/formaataanpassingen te selecteren.]D;]A;     - Tab voor een eenvoudige cyclus van zones en formaataanpassingen.]D;]A;     - Delete om de formaataanpassing in focus te verwijderen.]D;]A;     - Pijlen om de formaataanpassing in focus te verplaatsen.]]></Val>
```

Comment:
The word (EN) "cycle" is literally translated as a noun, but it is a verb, so the translation (NL) "cyclus" is wrong.

Reply:
Thank you for the feedback. Could you please create an issue for this and we'll bring it up with the loc team. Currently it's an external tool that creates these.